lib/web/apiserver.go

```diff
@@ -142,6 +144,13 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 		}
 	}
 
+	_, sshPort, err := net.SplitHostPort(cfg.ProxySSHAddr.String())
```

Comment:
Moved from `ProxyHostPort` to avoid re-parsing and repetitive error logs if the SSH address is invalid.